docs/content/getting-started/faq.md

```diff
@@ -0,0 +1,114 @@
+# FAQ
+
+## Why is Traefik Answering XXX HTTP Response Status Code ?
+
+Traefik is a dynamic reverse proxy,
+and while the documentation often demonstrates configuration options through file examples,
+the core feature of Traefik is its ability to be provided with configuration dynamically.
+
+The file provider allows providing [dynamic configuration](../configuration-overview/#the-dynamic-configuration) through a file or directory.
+This is different from the [static configuration](../configuration-overview/#the-static-configuration),
+which can also be provided by a file.
```

Comment:
Notably, a part of of the configuration is [static](../configuration-overview/#the-static-configuration), and can be provided by a file on startup, whereas various providers, such as the file provider, contribute dynamically all along the traefik instance lifetime to its [dynamic configuration](../configuration-overview/#the-dynamic-configuration) changes.